ex03.ex

```diff
@@ -101,7 +125,30 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([], []) do
```

Comment:
# 3

I think you'v made this too complicated. You can do the checking using pattern matching.

I was also a little confused by two functions called `list_equal` that do different things.

Here's my approach:

~~~ elixir
   def list_equal([], []), do: true
  def list_equal([h1], [h1]), do: true
  def list_equal([h1 | tail1], [h1 | tail2]), do: list_equal(tail1, tail2)
  def list_equal(a,b), do: false
~~~